src/accelerate/accelerator.py

```diff
@@ -2300,8 +2301,12 @@ def clip_grad_norm_(self, parameters, max_norm, norm_type=2):
                     xm.all_reduce("sum", gradients, scale=1.0 / self.num_processes)
                     # Set is_xla_gradients_synced to True to avoid all-reduce twice in the AcceleratedOptimizer step.
                     acc_opt.gradient_state.is_xla_gradients_synced = True
-        self.unscale_gradients()
```

Comment:
this two lines should not be deleted, the default behavior should be reserved

Reply:
done